### PR TITLE
ADEN-4685 Resolved state with two image assets

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/template/bfaaDesktop.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ define('ext.wikia.adEngine.template.bfaaDesktop', [
 ], function (uapContext,
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/template/bfaaDesktop.js:13]: MAJOR Function has 10 parameters which is greater than 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: extensions/wikia/AdEngine/js/spec/slot/resolvedState.spec.js [hunk 2]
```diff
@@ -105,21 +137,26 @@ describe('ext.wikia.adEngine.slot.resolveState', function () {
 			var rs = getModule();
 			mocks.qs.getVal.and.returnValue(testCase.queryParam);
 
-			expect(testCase.expected).toEqual(rs.setImage(testCase.params).backgroundImage.src);
+			if (testCase.twoAssets) {
```
Comment (mleczey) [inline extensions/wikia/AdEngine/js/spec/slot/resolvedState.spec.js]: Could you split this `if` to two different test cases?

File: extensions/wikia/AdEngine/js/slot/resolvedState.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	function isForcedByURLParam() {
+		return [true, 'true', '1'].indexOf(getQueryParam()) > -1;
+	}
+
+	function isBlockedByURLParam() {
+		return [false, 'blocked', 'false', '0'].indexOf(getQueryParam()) > -1;
+	}
+
+	function paramsAreCorrect (params) {
+		var correctSplitImages = params.resolvedState.leftImageSrc && params.resolvedState.rightImageSrc;
```
Comment (mleczey) [inline extensions/wikia/AdEngine/js/slot/resolvedState.js]: Just asking, didn't we want to resign from left/right naming? (use first, second, and give parameter which of those will have video overlay)

**Review by mleczey — CHANGES_REQUESTED** [review]

**Review by mleczey — APPROVED** [review]

**Pull request merged**